### PR TITLE
disable multi form submissions on password reset via token

### LINKED ISSUE #1153: Redirect / change page content after successul password reset
Linked to https://github.com/zooniverse/Panoptes/issues/1106

Seems some users are submitting the password reset form multipe times and it is invalid after the first submission as it has a one time use token. Best to notify success of the reset and disable the form.

### PR DESCRIPTION
**Pull request by camallen**
Closes #1153 - stop the one time token being used multiple times and provide better user messaging around the sequence of events when resetting the password.

### CODE REVIEW

File: app/pages/reset-password.cjsx [hunk 1]
```diff
@@ -102,10 +105,7 @@ module.exports = React.createClass
             <input ref="email" type="email" required onChange={@handleEmailChange} className="standard-input" defaultValue={@props.query?.email} size="50" />
           </p>
           <p>
-            {if @state.emailIsValid
-              <button type="submit" className="standard-button">Submit</button>
-            else
-              <button type="submit" className="standard-button" disabled>Submit</button>}
+            <button type="submit" className="standard-button" disabled={!@state.emailIsValid}>Submit</button>}
```
Comment (chrissnyder) [inline app/pages/reset-password.cjsx]: Hanging `}`

**Comment by chrissnyder** [comment]
I'm getting the mentioned 422 errors when I try to reset my password.

Steps:

1. Goto `http://localhost:3735/#/reset-password` and enter email and submit
2. Open email, grab token.
3. Goto `http://localhost:3735/#/reset-password?reset_password_token=TgqPd...`, enter new password.
4. Error: http://i.imgur.com/Sv5dvtv.png

**Comment by camallen** [comment]
Hmm - just used the steps above locally and it worked ok. Can you try with the updated PR?

screen shot 2015-07-21 at 13 49 14

**Comment by chrissnyder** [comment]
Still no dice. Username chrissnyder-panoptes with email cs@zoonivsere.org if that helps at all.

**Comment by chrissnyder** [comment]
This works on a couple new test accounts, so it's definitely something wrong with my dev account. Merging in and I'll look into my account separately.

**Comment by camallen** [comment]
Sorry - didn't get to this today. Let me know if I can help on this.

**Comment by edpaget** [comment]
It could be your dev accoun't have the unsubscribe token set.

**Pull request merged**